SpatialGDK/Source/SpatialGDK/Private/Tests/SpatialView/ViewDeltaTest.cpp

```diff
@@ -1,7 +1,134 @@
 // Copyright (c) Improbable Worlds Ltd, All Rights Reserved
 
+#include "OpsUtils.h"
+#include "SpatialViewUtils.h"
 #include "Tests/TestDefinitions.h"
 
 #include "SpatialView/ViewDelta.h"
 
 #define VIEWDELTA_TEST(TestName) GDK_TEST(Core, ViewDelta, TestName)
+
+using namespace SpatialGDK;
+
+VIEWDELTA_TEST(GIVEN_empty_view_WHEN_add_remove_THEN_get_empty_view_and_delta)
+{
+	EntityView InputView;
+	ViewDelta InputDelta;
+
+	TArray<Worker_Op> InputOpList = {
+		CreateAddEntityOp(2),
```

Comment:
magic numbers (here and elsewhere)- extract the entity ID out and have it as a constant throughout the file or per test (same with component ID and values, anything which is used more than once)